decidim-proposals/app/services/decidim/proposals/proposal_search.rb

```diff
@@ -26,11 +40,16 @@ def search_origin
         end
       end
 
-      # Handle the search_text filter
-      def search_search_text
-        query
-          .where("title ILIKE ?", "%#{search_text}%")
-          .or(query.where("body ILIKE ?", "%#{search_text}%"))
+      # Handle the activity filter
+      def search_activity
```

Comment:
Because "voted" is an option of the activity filter. In the future there will be more options like "created", "commented", etc. See http://decidim-design.herokuapp.com/proposals/